app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -104,7 +104,7 @@
           m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
           })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-          ga('create', 'UA-26179049-6', 'alphagov.co.uk');
+          ga('create', 'UA-26179049-6', '<%= ENV.fetch('GOVUK_APP_DOMAIN') %>');
```

Comment:
Only if you're running in Rails in production mode - line 100 checks `if Rails.env.production?`